### PR TITLE
[dmd-cxx] Fix Issue 17844 - std.process.execute should allow not capturing stderr 

### PR DESCRIPTION
**Pull request by ibuclaw**
From #5742

### CODE REVIEW

**Review by CyberShadow — APPROVED** [review]

File: std/process.d [hunk 1]
```diff
@@ -2549,6 +2569,10 @@ private auto executeImpl(alias pipeFunc, Cmd, ExtraPipeFuncArgs...)(
     auto r3 = executeShell("exit 123");
     assert(r3.status == 123);
     assert(r3.output.empty);
+    auto r4 = executeShell("echo stderr test, please ignore 1>&2",
```
Comment (berni44) [inline std/process.d:2572]: Couldn't this be solved similar to #7641?

Comment (ibuclaw) [reply std/process.d]: Now you tell me... :-)

Done.

**Comment by ibuclaw** [comment]
RC is imminent, so getting this in now. :-)

**Comment by ibuclaw** [comment]
Auto-merge toggled on

**Pull request merged**